precompiles/referenda/src/lib.rs

```diff
@@ -304,27 +305,27 @@ where
 			hash: proposal_hash,
 			len: proposal_len,
 		};
+		handle.record_log_costs_manual(2, 32 * 2)?;
```

Comment:
why is `topics: usize = 2` but for others further below it is  `topics: usize = 1`

please remind me what is that parameter?

Reply:
The logs below only contain 1 topic. The parameter refers to the number of topics included in a given log.